### ISSUE TITLE
Relationship FU Plan: Linda Rutherford

### ISSUE THREAD

**Issue by jaahmuhl** [promazo/100k-Mentor-Challenge-Outreach#145]
**User Story / Problem / Details**
The Content Editor is in charge of ensuring the initial relationship we foster with an interviewee does not decay over time for interviewees that we designate as important. As a result we want to follow a process that allows us to interact and build a relationship with the interviewee over time. The final step of the process should be focused on how connect with an interviewee over multiple interactions and over the course of a year.

**Technical Definition of Done**
Complete all steps in the process

**Definition of Ready**
All of the steps in the process have been approved by Brett 

**Process**
- [ ] Review the interview in its entirety 
- [ ] Thank you note to the participant
- [ ] Quick clip (30s) that shows the interviewee some of the raw footage/a particularly interesting part of the discussion
- [ ] ID potential contacts
- [ ] Formulate ask
- [ ] Create a Relationship Action Plan

**Comment by howie-emond** [comment]
Hi Linda, we wanted to reach out and thank you again for participating in our Executive Interview Series. 

During our discussion, we had the privilege of hearing about how you found Southwest, your values in an organization, growing as a leader and creating a personal brand, the future of opportunities in PR and Communications, and so much more. Both the Promazo team and our student audience will be excited to learn from you and your experiences.

Attached is a short clip from the interview that we found particularly interesting. The points you made on the acceptance of vulnerability, especially for younger generations, was insightful and will help our students immensely.

We are curious to hear if there are any opportunities to conduct similar interviews with some of your colleagues. We have identified the following as potential targets for future interviews: 
Michael G. Van de Ven
Robert E. Jordan
Tammy Romo
Andrew Watterson

Any contacts for these individuals, or other recommendations you may have, would benefit our project immensely.

Also, if you have any time in the future, we would be thrilled to have you on for a second interview, as your input and advice has been greatly appreciated. A discussion which dives further into your personal journey and visions on the industry’s future would be meaningful to our students.

Again, we thank you for your time and meaningful conversation. We look forward to building a closer relationship with you.

**Comment by howie-emond** [comment]
ATTATCHED CLIP
- In Rutherford interview folder
- linda-rutherford-follow-up.mov

**Comment by howie-emond** [comment]
Profile Links

Michael G. Van de Ven
(no linkedin)

Bob Jordan
https://www.linkedin.com/in/bob-jordan-25352b18/

Tammy Romo
(no linkedin)

Andrew Watterson
https://www.linkedin.com/in/andrew-watterson-8a5123/

**Comment by howie-emond** [comment]
UPDATED MESSAGE

Hi Linda, we wanted to reach out and thank you again for participating in our Executive Interview Series.

During our discussion, we had the privilege of hearing about how you found Southwest, your values in an organization, growing as a leader and creating a personal brand, the future of opportunities in PR and Communications, and so much more. Both the Promazo team and our student audience will be excited to learn from you and your experiences.

Attached is a short clip from the interview that we found particularly interesting. The points you made on the acceptance of vulnerability, especially for younger generations, was insightful and will help our students immensely.

We closed off our interview by introducing you to our mentorship program, and would like you to be involved. When your schedule permits, a 15 minute discussion to get your advice regarding our efforts would be greatly appreciated. Again, we thank you for your time and meaningful conversation and look forward to speaking with you soon.

(Same Clip)
